src/libs/NumberCovert.js

```diff
@@ -0,0 +1,57 @@
+import lodashGet from 'lodash/get';
+import loadashTrim from 'lodash/trim';
+import loadashTrimStart from 'lodash/trimStart';
+import loadashIncludes from 'lodash/includes';
+import loadashStartsWith from 'lodash/startsWith';
+import loadashPadStart from 'lodash/padStart';
+import translations from '../languages/translations';
+
+/**
+ * Returns a locally converted phone number without the country code
+ *
+ * @param {String} locale eg 'en', 'es-ES'
+ * @param {String} number
+ * @returns {string}
+ */
+function toLocalPhone(locale, number) {
+    const numString = loadashTrim(number);
+    const withoutPlusNum = loadashIncludes(numString, '+') ? loadashTrimStart(numString, '+') : numString;
+    const fullLocale = lodashGet(translations, locale, {});
+    const country = fullLocale.phoneCountryCode;
```

Comment:
I think this is equivalent and valid, right?
```suggestion
    const country = lodashGet(translations, [locale, 'phoneCountryCode']);
```

Reply:
Okay. I will use the suggested loadash method.